src/PrestaShopBundle/Form/Admin/Configure/ShopParameters/ProductPreferences/PageType.php

```diff
@@ -114,6 +114,23 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'Admin.Shopparameters.Help'
                 ),
                 'required' => false,
+            ])
+            ->add('feature_values_order', ChoiceType::class, [
+                'label' => $this->trans(
+                    'Order of feature values',
+                    'Admin.Shopparameters.Feature'
+                ),
+                'help' => $this->trans(
+                    'In the product page, feature values can be sorted alphabetically by value or by position.',
+                    'Admin.Shopparameters.Help'
+                ),
+                'choices' => [
+                    'Alphabetical' => 0,
+                    'Position' => 1,
```

Comment:
Can you use string values instead please? so we understand the meaning of each config
If you prefer to keep the int value it's fine but then we'd need some const matching these values for better understanding (which is why I suggested the string instead: `name`, `position`)